c7n/resources/ec2.py

```diff
@@ -1100,7 +1100,7 @@ class SetInstanceProfile(BaseAction, StateTransitionFilter):
         policies:
           - name: set-default-instance-profile
             resource: ec2
-            query:
+            filter:
```

Comment:
actually this needs to be filters and the k: v needs to be a list element.

Reply:
Wow can't believe I did that.......Such an easy fix and I mess it up